examples/django/oauth_app/slack_datastores.py

```diff
@@ -145,6 +139,24 @@ def find_installation(
 
         if len(rows) > 0:
             i = rows[0]
+            if user_id is not None:
+                # Fetch the latest bot token
```

Comment:
I suggest we add a bit more context to this comment to help remind future maintainers about this workaround.

```suggestion
                # Above, when fetching applicable installations, we retrieved relevant installations but filtered on matching user ID
                # In the case that this app uses both bot and user scopes, we also now fetch installations that match only on team/enterprise ID and contain a bot token
                # If any such results are retrieved, we merged them into the user-id-filtered results.
                # This is a fix for the situation described in https://github.com/slackapi/bolt-python/issues/664
```